src/sass/modules/_va-pagination.scss

```diff
@@ -5,80 +5,90 @@
   border-top: 1px solid $color-gray-lightest;
   font-size: .9em;
   overflow: hidden;
-  padding: 1rem;
+  padding: 2rem 0 1rem;
   position: relative;
   text-align: center;
   width: 100%;
 
-  a {
-    line-height: 2;
-    text-decoration: none;
+  > * {
+    display: inline-block;
+  }
+
+  &-prev {
+    @media (max-width: $medium-screen) {
+      margin-left: -3rem;
+      padding: 0 1rem 0 0;
+    }
+
+    @media (min-width: $medium-screen) {
+      padding: 0 1rem;
+    }
+
+    &:not(:empty) {
+      @media (min-width: $small-screen) {
+        &::before {
+          content: '\003c\a0\a0';
+        } 
+      }
+    }
   }
 
-  &-prev,
   &-next {
-    padding: 0 1rem;
+    @media (max-width: $medium-screen) {
+      margin-right: -3rem;
+      padding: 0 0 0 1rem;
+    }
+
+    &:not(:empty) {
+      @media (min-width: $small-screen) {
+        &::after {
+          content: '\a0\a0\003e';
+        }
+      }
+    }
+  }
+
+  &-prev,
+  &-next {    
+    position: relative;
+    top: -1rem;
```

Comment:
Can remove these two lines by adding `vertical-align: top` to the `> *` selector.